chain/syncstate.go

```diff
@@ -32,12 +32,20 @@ type SyncerState struct {
 }
 
 func (ss *SyncerState) SetStage(v api.SyncStateStage) {
+	if ss == nil {
+		return
```

Comment:
log.Warn in these?

Reply:
ehhhh, the point here is for things like tests to be able to run sync tests without having to inject a syncerstate tracker